dev/dll/XamlControlsResources.cpp

```diff
@@ -74,6 +78,13 @@ void XamlControlsResources::UpdateSource()
 
             hstring releasePrefix;
 
+#ifdef USE_INSIDER_SDK
```

Comment:
Remove `#ifdef USE_INSIDER_SDK`
no matter if USE_INSIDER_SDK or not, we should have 21h1_themeresources.xaml/21h1_generic.xaml and compact...